extensions/content-usage-metrics/manifest.json

```diff
@@ -3083,25 +3115,28 @@
   },
   "files": {
     "app.R": {
-      "checksum": "9503ad4badfebc475f44466b7d7c8ef7"
+      "checksum": "93046fabfbe4c7e1e1360ba431583cf0"
     },
     "get_usage.R": {
-      "checksum": "5b2476731ce8e473c09abd1c593ccfa7"
+      "checksum": "f232066323b2f15d618663c2c2e97d0a"
+    },
+    "www/styles.css": {
+      "checksum": "86129d1eb71c75924c7584af191c4cd9"
     }
   },
   "users": null,
   "extension": {
-    "name": "content-usage-metrics-dashboard",
-    "title": "Content Usage Metrics Dashboard",
-    "description": "See what content is popular on your Connect instance. Dive into details for a content item to learn who's viewing it, and when.",
-    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/who-is-visiting-this-content",
-    "tags": ["R", "shiny", "usage-metrics"],
+    "name": "content-usage-metrics",
+    "title": "Content Usage Metrics",
+    "description": "understand content usage patterns on your Connect server. Browse an overview of usage across your published content, and dive into a detailed breakdown of visit-level data for each piece of content.",
+    "homepage": "https://github.com/posit-dev/connect-extensions/tree/main/extensions/content-usage-metrics",
+    "tags": [],
     "minimumConnectVersion": "2025.04.0",
```

Comment:
We need to add `requiredFeatures` since this requires "OAuth Integrations" to function

Reply:
Done